next.config.js

```diff
@@ -2,3 +2,17 @@
 const nextConfig = {}
 
 module.exports = nextConfig
+
+// Everything below *needs* to be part of the default template that running npm create cloudflare@latest gives you
+// It needs to be documented in real dev docs
```

Comment:
This has to be loud, at the top of any getting started path we have (since many people will have existing Next.js apps, or be creating them outside of `npm create cloudflare@latest`, outside of dash.